libs/components/core/src/lib/modules/adapter-service/adapter.service.ts

```diff
@@ -125,10 +125,14 @@ export class SkyCoreAdapterService {
    * recieve focus if no focusable children are found.
    */
   public getFocusableChildrenAndApplyFocus(
-    elementRef: ElementRef,
+    elementRef: ElementRef | undefined,
     containerSelector?: string,
     focusOnContainerIfNoChildrenFound = false
   ): void {
+    if (!elementRef) {
```

Comment:
Instead of making this change in core, could we just add an `if` check to the popovers component?
```
public applyFocus(): void {
  if (this.popoverRef && this.isOpen) {
    this.#coreAdapterService.getFocusableChildrenAndApplyFocus(
      ...
```